ci-operator/step-registry/telcov10n/functional/cnf-ran/send-slack-notification/telcov10n-functional-cnf-ran-send-slack-notification-commands.sh

```
@@ -43,38 +41,42 @@ BASTION_USER=$(cat /var/group_variables/common/all/ansible_user)
-if scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i /tmp/temp_ssh_key "${BASTION_USER}@${BASTION_IP}":/tmp/.polarion_url "${SHARED_DIR}/polarion_url" 2>/dev/null; then
-  if [[ -f "${SHARED_DIR}/polarion_url" ]] && [[ -s "${SHARED_DIR}/polarion_url" ]]; then
-    POLARION_URL="$(cat ${SHARED_DIR}/polarion_url)"
-    echo "Polarion URL: ${POLARION_URL}"
-  fi
-else
-  echo "No polarion URL file found on bastion"
-fi
+SSH_OPTS="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i /tmp/temp_ssh_key"
 
 REPORT_FLAGS=""
 for REPORTPORTAL_FILE in ${REPORTPORTAL_FILES//;/ }; do
```

Shellcheck (0.11.0)

[info] Possible misspelling: REPORTPORTAL_FILES may not be assigned. Did you mean REPORTPORTAL_FILE?

(SC2153)